operations/libero.md

```diff
@@ -0,0 +1,47 @@
+# Libero howto
+
+The Libero is the person handling 1st level support and on-call work during business hours. The Libero’s main duty is to provide swift help for our customers and us.
+
+This is the description what we expect the Libero to do and how to handle incoming requests.
+
+## Start of Libero
+
+1. Check #support for open 2nd level requests to make sure nothing is overseen.
+2. Trawl the ticket system for "orphaned", non-Libero tickets, post in #support as short (delete ticket-expansions visual after creation) to remind assignee.
+3. Check Intercom for unclosed requests. 
+
+## End of Libero
+1. Add a summary as an internal comment in tickets where next step is not 100% obvious to guide next Libero.
+2. Assign all tickets to next Libero. Do not hold on to "small" stuff.
+3. Check PagerDuty so on-call doesn’t get surprise incidents. Hand them over with a comment in #on-call.
+
+## General guidance
```

Comment:
Since this doesn't apply only to the Libero but to everyone doing support, I think that it makes sense to put general support behaviour on a separate page. This page could later be extended with topics like wording recommendations and other things.

Reply:
I agree. I'd say we'd move this out of this document later on. For now the main support person has everything at hand in here.